dashboard/db/schema.rb

```diff
@@ -1581,19 +1581,19 @@
     t.index ["user_id"], name: "index_survey_results_on_user_id", using: :btree
   end
 
-  create_table "teacher_feedbacks", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
-    t.text     "comment",                  limit: 65535
+  create_table "teacher_feedbacks", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci" do |t|
+    t.text     "comment",                  limit: 16777215
```

Comment:
Did the size of the text column increase due to the change in charset?

Reply:
It does for me when I try this locally. Furthermore, when I roll the migration forward, back, and then forward, I get an even bigger size:
![Screen Shot 2020-10-28 at 10 46 06 AM](https://user-images.githubusercontent.com/8001765/97475800-ec508780-190a-11eb-849b-8346b4d3093e.png)

What do folks think about adding something to the `down` clause of the migration to restore the original column size?
```
change_column :teacher_feedbacks, :comment, :text, limit: 65535
```

Reply:
or, maybe it would make more sense to restore the original column width as part of the `up` clause? it seems like the odds of having 64KB of teacher feeback are pretty slim.